Appveyor fails to pack Redis

AppVeyor fails on 'dotnet pack' only against [Redis project](https://ci.appveyor.com/project/Greentube/monitoring/branch/master/artifacts).

`
C:\Program Files\dotnet\sdk\1.0.4\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(104,5): error MSB4018: The "PackTask" task failed unexpectedly. [C:\projects\monitoring\src\Greentube.Monitoring.Redis\Greentube.Monitoring.Redis.csproj]
C:\Program Files\dotnet\sdk\1.0.4\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(104,5): error MSB4018: System.IO.FileNotFoundException: File not found: 'C:\projects\monitoring\src\Greentube.Monitoring.Redis\bin\Release\net46\Greentube.Monitoring.Redis.dll'. [C:\projects\monitoring\src\Greentube.Monitoring.Redis\Greentube.Monitoring.Redis.csproj]
C:\Program Files\dotnet\sdk\1.0.4\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(104,5): error MSB4018:    at NuGet.Packaging.PackageBuilder.AddFiles(String basePath, String source, String destination, String exclude) [C:\projects\monitoring\src\Greentube.Monitoring.Redis\Greentube.Monitoring.Redis.csproj]
C:\Program Files\dotnet\sdk\1.0.4\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(104,5): error MSB4018:    at NuGet.Packaging.PackageBuilder.PopulateFiles(String basePath, IEnumerable`1 files) [C:\projects\monitoring\src\Greentube.Monitoring.Redis\Greentube.Monitoring.Redis.csproj]
C:\Program Files\dotnet\sdk\1.0.4\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(104,5): error MSB4018:    at NuGet.Commands.MSBuildProjectFactory.CreateBuilder(String basePath, NuGetVersion version, String suffix, Boolean buildIfNeeded, PackageBuilder builder) [C:\projects\monitoring\src\Greentube.Monitoring.Redis\Greentube.Monitoring.Redis.csproj]
C:\Program Files\dotnet\sdk\1.0.4\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(104,5): error MSB4018:    at NuGet.Commands.PackCommandRunner.BuildFromProjectFile(String path) [C:\projects\monitoring\src\Greentube.Monitoring.Redis\Greentube.Monitoring.Redis.csproj]
C:\Program Files\dotnet\sdk\1.0.4\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(104,5): error MSB4018:    at NuGet.Commands.PackCommandRunner.BuildPackage(String path) [C:\projects\monitoring\src\Greentube.Monitoring.Redis\Greentube.Monitoring.Redis.csproj]
C:\Program Files\dotnet\sdk\1.0.4\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(104,5): error MSB4018:    at NuGet.Commands.PackCommandRunner.BuildPackage() [C:\projects\monitoring\src\Greentube.Monitoring.Redis\Greentube.Monitoring.Redis.csproj]
C:\Program Files\dotnet\sdk\1.0.4\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(104,5): error MSB4018:    at NuGet.Build.Tasks.Pack.PackTask.Execute() [C:\projects\monitoring\src\Greentube.Monitoring.Redis\Greentube.Monitoring.Redis.csproj]
C:\Program Files\dotnet\sdk\1.0.4\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(104,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [C:\projects\monitoring\src\Greentube.Monitoring.Redis\Greentube.Monitoring.Redis.csproj]
C:\Program Files\dotnet\sdk\1.0.4\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(104,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__25.MoveNext() 
`

[Issue raised with AppVayor](https://github.com/appveyor/ci/issues/1767)

--- a/src/Greentube.Monitoring.Redis/Greentube.Monitoring.Redis.csproj
+++ b/src/Greentube.Monitoring.Redis/Greentube.Monitoring.Redis.csproj
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;net46</TargetFrameworks>
+    <TargetFrameworks>netstandard1.5</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <AssemblyName>Greentube.Monitoring.Redis</AssemblyName>
     <PackageVersion>0.0.0</PackageVersion>
@@ -10,8 +10,7 @@
     <Description>Redis health monitor for Greentube.Monitoring allowing probes and health check of Redis databases</Description>
     <PackageTags>redis;health-checks;monitor;health;load-balancer</PackageTags>
     <PackageProjectUrl>https://github.com/Greentube/monitoring</PackageProjectUrl>
-    <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
-    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.3' ">$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
@@ -23,7 +22,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Coverage|AnyCPU'">
     <DebugType>full</DebugType>
   </PropertyGroup>
-  
+
   <ItemGroup>
     <Compile Include="..\..\GlobalAssemblyInfo.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" />
   </ItemGroup>
@@ -32,14 +31,9 @@
     <ProjectReference Include="..\Greentube.Monitoring\Greentube.Monitoring.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
-    <PackageReference Include="StackExchange.Redis" Version="1.1.604-alpha" />
-  </ItemGroup>
-
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net46' ">
-    <PackageReference Include="StackExchange.Redis" Version="1.0.488" />
-    <Reference Include="System" />
-    <Reference Include="Microsoft.CSharp" />
+  <ItemGroup>
+    <PackageReference Include="StackExchange.Redis" Version="1.2.6" />
   </ItemGroup>
 
 </Project>
+

